helix-term/src/commands.rs

```diff
@@ -3325,8 +3326,8 @@ fn buffer_picker(cx: &mut Context) {
             .map(|(_, doc)| new_meta(doc))
             .collect(),
         BufferMeta::format,
-        |editor: &mut Editor, meta, action| {
-            editor.switch(meta.id, action);
+        |cx, meta, _action| {
+            cx.editor.switch(meta.id, Action::Replace);
```

Comment:
This should use `action`, looks like it got reverted as part of conflict resolution.

Reply:
Fixed, sorry.